### PR TITLE
issue208-get-windows-builds-working

### PR DESCRIPTION
**Pull request by JEStaubach**
appveyor.yml
    - include msys\bin to provide make
    - no command for bdist_wheel when running python setup.py so replace with bdist
Makefile
    - windows would not allow the complex python -c commands so replaced with a temp script.

Testing - Already had failed tests that are passing now. No new tests seemed to be required.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ test_script:
   - "%PYTHON%/Scripts/tox -e %TOX_ENV%"
 
 after_test:
-  - "%PYTHON%/python setup.py bdist_wheel"
+  - "%PYTHON%/python setup.py bdist"
```
Comment (eliasdorneles) [inline appveyor.yml]: Hm, I think there is no point in distributing a binary if it's not wheel.
Do you know why it's failing to build a wheel in appveyor, even the wheel format package installs fine?

File: {{cookiecutter.project_slug}}/Makefile [hunk 2]
```diff
@@ -11,20 +11,17 @@ webbrowser.open("file://" + pathname2url(os.path.abspath(sys.argv[1])))
...
 
 help:
-	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
+	echo "import re, sys" > .print_help_script.py
+	echo "for line in sys.stdin:" >> .print_help_script.py
+	echo "    match = re.match(r'^([a-zA-Z_-]+):.*?## (.*)$$', line)" >> .print_help_script.py
+	echo "    if match:" >> .print_help_script.py
+	echo "        target, help = match.groups()" >> .print_help_script.py
+	echo "        print('%-20s %s' % (target, help))" >> .print_help_script.py
+	@python .print_help_script.py < $(MAKEFILE_LIST)
```
Comment (eliasdorneles) [inline {{cookiecutter.project_slug}}/Makefile]: Is this change really necessary?
Does this fail on Windows in general or only in the Appveyor build?

Comment (JEStaubach) [inline {{cookiecutter.project_slug}}/Makefile]: I really hope not, there has to be a better way, I just couldn't work out how. You are right, looks like only an issue in Appveyor build. I'll take another look.

**Comment by eliasdorneles** [comment]
Hey @JEStaubach, sorry for the delay, I hadn't seen these latest changes!
You did it, buddy, that was it!! Thank you so much!! :bow:
